server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,9 +290,37 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
-#buildoutput_pre{
+#buildoutput_pre {
     overflow: auto;
     white-space: pre-wrap;
+
+    padding-left: 10px;
+    padding-bottom: 10px;
+
+    @each $color in 'black', 'red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white', 'gray' {
```

Comment:
Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `yellow` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `magenta` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `cyan` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings